### ISSUE TITLE
Unable to upload images through API request

### ISSUE THREAD

**Issue by n0t4u** [DefectDojo/django-DefectDojo#4443]
**Bug description**
I´m trying to automate the upload of evidences in each finding using the V2 api. The request path is POST
​/findings​/{id}​/files​/. The problem is that I´m unable to upload images due to the encoding type of the form. I have tried both via web (/api/v2/doc/) and via cURL as you can see in the requests below.
```bash
└─$ curl -X POST "https://example.com/api/v2/findings/464/files/" -H  "accept: application/json" -H  "Content-Type: multipart/form-data" -H  "Authorization: Token TOKEN" -F "title"="XSS de prueba" -F "file='@/home/n0t4u/Descargas/XSS.jpeg'"
{"file":["The submitted data was not a file. Check the encoding type on the form."]}                                                                                                                                                                                                                                           
 
└─$ curl -X POST "https://example.com/api/v2/findings/464/files/" -H  "accept: application/json" -H  "Content-Type: multipart/form-data" -H  "Authorization: Token TOKEN" -F "title=XSS de prueba" -F "file='@/home/n0t4u/Descargas/XSS.jpeg'" 
{"file":["The submitted data was not a file. Check the encoding type on the form."]}                                                                                                                                                                                                                                                              
                                                                                                                 
└─$ curl -X POST "https://example.com/api/v2/findings/464/files/" -H  "Content-Type: multipart/form-data" -H  "Authorization: Token TOKEN" -F "title=XSS" -F "file=['@/home/n0t4u/Descargas/XSS.jpeg']" 
{"file":["The submitted data was not a file. Check the encoding type on the form."]} 
```
Several files have been tested and the error is the same always.
I have searched for this error and I think is a problem related with Django but, as I was unable to find where this request is handled in the server, I couldn´t move forward.     

**Steps to reproduce**
1. Send a POST request as is shown in /api/v2/doc/

**Expected behavior**
I expect to upload the file (in this case images) to the finding I pass as a parameter in the URL

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [X] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: Linux
 - DefectDojo Commit Message: [2021-02-05 22:45:34 +0100] f567f53a: Merge pull request #3806 from madchap/fix-diagram-master [ (HEAD -> master, origin/master, origin/HEAD)]

**Screenshots** (optional)
Here is an image with the error when the request is done through a browser
![API request](https://user-images.githubusercontent.com/71253024/116995595-8a6a4380-acda-11eb-843d-3e29c1764d46.png)

**Comment by valentijnscholten** [comment]
@Maffooch Did you manage to find a way to submit the files during your PR testing?

**Comment by n0t4u** [comment]
Hi, have you been able to find the reason of this issue?

**Comment by valentijnscholten** [comment]
#5509

**Comment by StefanFl** [comment]
merged for release 2.5.0
